org.eclipse.scout.rt.client/src/main/java/org/eclipse/scout/rt/client/ui/basic/table/TableEventBuffer.java

```diff
@@ -585,6 +590,12 @@ public void complete() {
       m_mergedRows.addAll(m_targetRows);
       m_targetEvent.setRows(m_mergedRows);
       m_mergedRows = null;
+
+      m_mergedUpdatedColumns.putAll(m_targetUpdatedColumns);
```

Comment:
Is this necessary? The map of updatedColumns on the target event will not be replace entirely (as with the other two lists), but only agumented.

If not necessary, do we even need the m_targetUpdatedColumns member?